hudi-utilities/src/main/java/org/apache/hudi/utilities/sources/KafkaSource.java

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
Created a common base class for KafkaSources since there was a lot of repeated code between json, avro, and now proto implementations